README.md

```diff
@@ -53,18 +53,39 @@ Now that we've covered the basics of using `dotnew new`, lets move on to info fo
 If you're authoring templates, or interested in contributing to this repo, then you're likely interested in how to use the latest version of this experience.
 The steps required are outlined below.
 
-Step 1: Get the SDK for your platform from [dotnet/cli](https://github.com/dotnet/cli)
+## Prerequisites
+In order to work with this project, you will need the following.
+- [Visual Studio 2017](https://www.visualstudio.com/)
 
-Step 2: Clone this repository to your local machine
+## Setup
 
-Step 3:  Run the setup script
- - **Windows:** [setup.cmd](https://github.com/dotnet/templating/blob/master/setup.cmd)
- - **Mac/Linux**: [setup.sh](https://github.com/dotnet/templating/blob/master/setup.sh) 
+Step 1:
+- Get the SDK for your platform from [dotnet/cli](https://github.com/dotnet/cli).
+- Typically, development is  done against the latest RC FTS Release.  
+  - It is best to ask the team to clear up any compatibility concerns before starting.
```

Comment:
Am I correct in stating that acquiring the SDK is irrelevant to building and running a 'dotnet new3' execution environment? And that acquiring the SDK only comes into play when actually doing development.  Which by way of, would make this step more relevant to the "debugging" section.